test/spec/GeoExt/data/store/Layers.test.js

```diff
@@ -6,5 +6,141 @@ describe('GeoExt.data.store.Layers', function() {
         it('is defined', function() {
             expect(GeoExt.data.store.Layers).not.to.be(undefined);
         });
+
+        describe('constructor', function() {
+            it('throws if no map component provided', function() {
+                expect(function() {
+                    Ext.create('GeoExt.data.store.Layers');
+                }).to.throwException();
+            });
+        });
+    });
+
+    describe('function "bindMap"', function() {
+        var store;
+        var map;
+        beforeEach(function() {
+            var layer = new ol.layer.Vector();
+            map = new ol.Map({
+                layers: [layer]
+            });
+            store = Ext.create('GeoExt.data.store.Layers', {map: map});
+        });
+
+        it('exists', function() {
+            expect(store.bindMap).not.to.be(undefined);
+        });
+
+        it('applies the map correctly', function() {
+            expect(store.map).to.be(map);
+        });
+    });
+
+    describe('function "unbindMap"', function() {
+        var store;
+        beforeEach(function(done) {
+            var layer = new ol.layer.Vector();
+            var map = new ol.Map({
+                layers: [layer]
+            });
+            store = Ext.create('GeoExt.data.store.Layers', {map: map});
+
+            done();
+        });
+
+        it('exists', function() {
+            expect(store.unbindMap).not.to.be(undefined);
+        });
+
+        it('removes the map', function() {
+            expect(store.map).to.be.null;
```

Comment:
Did you mean to write `to.be(null)`?

And how would that pass?  Don't you need to call `unbindMap` (or do sth. that then calls `unbindMap`)?